tests/test_sn.py

```diff
@@ -97,20 +97,20 @@ def test_getpdu_xcvr_info(self):
 
     def test_getpdu_xcvr_dom(self):
         expected_mib = {
-            1000 * 1 + 1: "DOM Temperature Sensor for Ethernet0",
-            1000 * 1 + 2: "DOM Voltage Sensor for Ethernet0",
-            1000 * 1 + 11: "DOM RX Power Sensor for Ethernet0/1",
-            1000 * 1 + 21: "DOM RX Power Sensor for Ethernet0/2",
-            1000 * 1 + 31: "DOM RX Power Sensor for Ethernet0/3",
-            1000 * 1 + 41: "DOM RX Power Sensor for Ethernet0/4",
-            1000 * 1 + 12: "DOM TX Bias Sensor for Ethernet0/1",
-            1000 * 1 + 22: "DOM TX Bias Sensor for Ethernet0/2",
-            1000 * 1 + 32: "DOM TX Bias Sensor for Ethernet0/3",
-            1000 * 1 + 42: "DOM TX Bias Sensor for Ethernet0/4",
-            1000 * 1 + 13: "DOM TX Power Sensor for Ethernet0/1",
-            1000 * 1 + 23: "DOM TX Power Sensor for Ethernet0/2",
-            1000 * 1 + 33: "DOM TX Power Sensor for Ethernet0/3",
-            1000 * 1 + 43: "DOM TX Power Sensor for Ethernet0/4",
+            1000 * 1 + 1: "DOM Temperature Sensor for etp1",
+            1000 * 1 + 2: "DOM Voltage Sensor for etp1",
+            1000 * 1 + 11: "DOM RX Power Sensor for etp1/1",
```

Comment:
according to the naming convention it should be etp1a, etp1b, etp1c, etp1d

Reply:
https://github.com/Azure/SONiC/blob/master/doc/sonic-port-name.md

Reply:
According to requirement doc for SNMP transceiver sensor monitoring the format  is "alias/lane_number".
Not all platforms currently have aliases in new naming convention, so the alias will be the name "EthernetN".
Also, even if we have a split port 'etp1a', the MIB table for 'etp1a' will contain sensor info for all 4 lanes, so it is necessary to specify lane number in description.
I can change this to "alias (lane X)" - that may be less confusing ?.